docs/topics/what-is-the-toolkit.adoc

```diff
@@ -11,14 +13,21 @@
 [discrete]
 == What is the {ProductName}?
 
-The {ProductName} ({ProductShortName}) is an extensible and customizable rule-based tool that simplifies the migration of Java applications.
+The {ProductName} ({ProductShortName}) is an extensible and customizable rule-based tool that simplifies the migration and modernization of Java applications.
 
-The {ProductShortName} examines application artifacts, including project source directories and application archives, then produces an HTML report that highlights areas needing changes. {ProductShortName} can migrate Java applications from earlier versions of Red Hat JBoss Enterprise Application Platform or from other application servers, such as Oracle WebLogic Server or IBM WebSphere Application Server.
+{ProductShortName} examines application artifacts, including project source directories and application archives, then produces an HTML report that highlights areas needing changes. {ProductShortName} supports many migration paths including:

```

Comment:
IBM 2021 (p. 153) says that leadin sentence for a list should be a full sentence.
Add something like "the following examples"

Reply:
"then" > "and then"

Reply:
"that highlights areas needing changes" > "highlighting areas needing changes"

Reply:
If you change "The {ProductShortName}" to just "{ProductShortName}", make sure that the change is consistent in all docs.